### ISSUE TITLE
config: no error message if config file pass the max length in hirte manager 

### ISSUE THREAD

**Issue by dougsland** [engelmi/hirte#431]
## Describe the bug

hirte should add a logic to not start and give an error (or at least a warning) message if the max length for the AllowedNodeName (**or any other option**) got achieved.

At this moment, if users are not aware of https://github.com/containers/hirte/blob/main/doc/docs/configuration.md#maximum-line-length it will **NOT** show any error on the hirte manager side just "anonymous node" tried to connect and in the node side "Unexpected node name". 

**Manager**
```
AllowedNodeNames=control,node1,qm.control123,node2,node3,node4,node5,node6,node7,node8,node9,node10,node11,node12,node13,node14,node15,node16,node17,node18,node19,node20,node21,node22,node23,node24,node25,node2
6

#  builddir/src/manager/hirte -c /etc/hirte/hirte.conf
2023-08-04 03:20:51,250+0000 DEBUG	: Final configuration used
2023-08-04 03:20:51,250+0000 INFO	: Starting hirte 0.5.0
2023-08-04 03:20:51,251+0000 DEBUG	: Connected to system bus
2023-08-04 03:20:54,795+0000 INFO	: Anonymous node disconnected
2023-08-04 03:20:57,045+0000 INFO	: Anonymous node disconnected
2023-08-04 03:20:59,295+0000 INFO	: Anonymous node disconnected
2023-08-04 03:20:59,733+0000 INFO	: Anonymous node disconnected
```

**Agent 1**:
```
NodeName=node25

# builddir/src/agent/hirte-agent -c /etc/hirte/agent.conf
ERROR	: Registering as 'node25' failed: Unexpected node name
```

Agent 02:
```
NodeName=control

# ./builddir/src/agent/hirte-agent -c /etc/hirte/hirte.conf
2023-08-04 03:22:30,444+0000 INFO	: Starting hirte-agent 0.5.0
2023-08-04 03:22:30,454+0000 INFO	: Connecting to manager on tcp:host=127.0.0.1,port=842
2023-08-04 03:22:30,455+0000 INFO	: Connected to manager as 'control'
```

If the user use the format below it start working:

```
AllowedNodeNames=control,
  node1,
  qm.control123,
  node2,
  node3,
  node4,
  node5,
  node6,
  node7,
  node8,
  node9,
  node10,
  node11,
  node12,
  node13,
  node14,
  node15,
  node16,
  node17,
  node18,
  node19,
  node20,
  node21,
  node22,
  node23,
  node24,
  node25,
  node26

Output from the node:
2023-08-04 03:23:35,718+0000 INFO	: Connected to manager as 'node25'
``` 

See Also: https://github.com/containers/hirte/pull/428

**Comment by dougsland** [comment]
Just an example in hirte manager:

```
# hirte -c /etc/hirte/hirte.conf
line too long in /etc/hirte/hirte.conf
cannot parse file: /etc/hirte/hirte.conf. See hirte documentation to learn how to  
increase the maximum size limit.
```

**Comment by rhatdan** [comment]
SGTM

**Comment by engelmi** [comment]
Good catch!
Should've been fixed in #433 - could you verify that? @dougsland You should get an error message like this:
```bash
Error parsing configuration file '<some-path>/hirte-default.conf' on line 13
```
__Note:__ `inih` only returns the line number where a parsing error occurred, e.g. line too long. Actually, its the erroneous line + 1. This also means that a more detailed error message is not possible, although having the line number already helps a lot, I think. 

__Note:__ I increased the maximum line length to ~500 (excluding key). Somehow that file slipped being committed in #428.
